### PR TITLE
Update MSP2TEXT command

### PR DESCRIPTION
**Pull request by haslinghuis**
Constants were wrongly part of `MSPCodes`.
Moved them to `MSPHelper` as they are only used there.

Naah ended up with including missing parts and rebasing functionality aiding future additions.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Review by sugaarK — APPROVED** [review]

**Pull request merged**
